crates/client/block_import/src/rayon.rs

```diff
@@ -19,26 +22,44 @@ impl Default for RayonPool {
 impl RayonPool {
     pub fn new() -> Self {
         let n_cores = thread::available_parallelism().expect("Getting the number of cores").get();
-
         let max_tasks = n_cores * 2;
```

Comment:
Just thinking loud here (and probably this isn't for this issue) - Juno has an interesting feature where I can set the max concurrent vms that will be used for `starknet_calls`. Playing around with that allowed use to configure Juno fro Kakarot in a different way than Starknet because Juno has a lot of heavy calls. Can the configuring `2` number here also help chains that have blocks of different sizes?

Reply:
If I am correct that might be a different problem entirely as the `RayonPool` is specifically not made to be used in a RPC context.

Reply:
I see but my point was more around configuring number of cores you use depending on the application needs. From what I understand, the `RayonPool` is used for block imports. So the number of block imports you want to work with concurrently might change depending upon how heavy the block is maybe? Anyways, this is not urgent and maybe I should find a breaking case and then we can add this :)

Reply:
@apoorvsadana 

> So the number of block imports you want to work with concurrently might change depending upon how heavy the block is maybe

That's a good question, but it's not relevant here - this is a failsafe that allows backpressure in the pipeline. It basically only helps to stop a DoS vector when we'll have p2p and people will be able to push blocks to us.
This should not affect the speed of the pipeline.

The question about tuning the numbers wrt. network speed, memory, size of blocks is higher up in the pipeline: we handle 10 fetches (including prevalidate which is class compilation, conversion & hashing) in parallel, and it's still in sync/l2.rs. It'll be moved to block import, someday:tm: and yea we forgot about setting a cli (or chainconfig?) param for it.